### PR TITLE
1007 - Bedspace usage report endpoint

### PR DESCRIPTION
**Pull request by brownbearsoftware**
https://trello.com/c/GQxvkTmU/1007-download-a-report-on-all-bookings-voids-and-turnaround-times

### CODE REVIEW

**Comment by brownbearsoftware** [comment]
[Download a report on all bookings, voids and turnaround times](https://trello.com/c/GQxvkTmU/1007-download-a-report-on-all-bookings-voids-and-turnaround-times)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/WorkingDayCountService.kt [hunk 1]
```diff
@@ -19,4 +19,24 @@ class WorkingDayCountService(
     }
     return from.getDaysUntilInclusive(to).filter { it.isWorkingDay(bankHolidays) }.size
   }
+
+  fun calculateEndDate(startDate: LocalDate, numberOfWorkingDays: Int): LocalDate {
```
Comment (tahb) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/WorkingDayCountService.kt:23]: @danlivings-dxw I wonder if this logic something you're also working on for booking creation.

**Review by tahb — APPROVED** [review]
From what I can tell this looks good. I can imagine that Dan has been doing similar things so there may be overlap and conflicts ahead.

**Comment by brownbearsoftware** [comment]
@tahb requested a rereview as made some small fixes around date ranges being inclusive + turnarounds starting the day after departure

**Review by danlivings-dxw — APPROVED** [review]
LGTM

**Pull request merged**